qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -506,7 +506,8 @@ def _table_name(cls, obj_id):
         return "%s%d" % (cls._table_prefix, obj_id)
 
     @classmethod
-    def _clean_validate_template(cls, md_template, study_id, restriction_dict):
+    def _clean_validate_template(cls, md_template, study_id, restriction_dict,
+                                 current_columns=None):
         """Takes care of all validation and cleaning of metadata templates
 
         Parameters
```

Comment:
In the Parameters section, I'd personally prefer to have `pd.DataFrame` instead of just `DataFrame`.
Again, this is just personal preference.